config/config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "logger": {
+        "Host": "localhost",
+        "port": 12201,
+        "Pass_Secret": "usermanagerlvl48",
+        "Pass_SHA2": "9cc1ee455a3363ffc504f40006f70d0c8276648a5d3eb3f9524e94d1b7a83aef",
```

Comment:
Its not a good idea to push files like this

Reply:
Malfunctioned .gitignore. Fixed